internal/controller/setup.go

```diff
@@ -94,10 +95,12 @@ func SetupManagers(mgr manager.Manager, userAgent, defaultProxyImage string) err
 
 // RegisterPodWebhook register the webhook to mutate pods
 func RegisterPodWebhook(mgr ctrl.Manager, u *workload.Updater) error {
+
```

Comment:
nit: does this whitespace need to be in the PR?

Reply:
Outdated. Merge error.